src/java.base/share/classes/java/lang/ProcessBuilder.java

```diff
@@ -1112,24 +1102,13 @@ private Process start(Redirect[] redirects) throws IOException {
             }
             return process;
         } catch (IOException | IllegalArgumentException e) {
-            String exceptionInfo = ": " + e.getMessage();
-            Throwable cause = e;
-            if ((e instanceof IOException) && security != null) {
-                // Can not disclose the fail reason for read-protected files.
-                try {
-                    security.checkRead(prog);
-                } catch (SecurityException se) {
-                    exceptionInfo = "";
-                    cause = se;
-                }
-            }
             // It's much easier for us to create a high-quality error
             // message than the low-level C code which found the problem.
             throw new IOException(
                 "Cannot run program \"" + prog + "\""
                 + (dir == null ? "" : " (in directory \"" + dir + "\")")
-                + exceptionInfo,
-                cause);
+                + e.getMessage(),
+                e);
```

Comment:
Can you check the format of the exception? It looks like original put ": " before the exception message. With the change it looks like it will immediately follow without spacing.